public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -39,11 +39,11 @@ interface Props {
 }
 
 export function LogsTable(props: Props) {
-  const { timeZone, splitOpen, range, logsSortOrder, width, logsFrames, columnsWithMeta } = props;
+  const { timeZone, splitOpen, range, logsSortOrder, width, logsFrame, columnsWithMeta } = props;
   const [tableFrame, setTableFrame] = useState<DataFrame | undefined>(undefined);
 
   // Only a single frame (query) is supported currently
-  const logFrameRaw = logsFrames ? logsFrames[0] : undefined;
+  const logFrameRaw = logsFrame ? logsFrame : undefined;
```

Comment:
Do we need this? I seems to be redundant as do then `we const logsFrame = logFrameRaw ? parseLogsFrame(logFrameRaw) : undefined;` couple lines bellow.

Reply:
Also I don't think that comment above this is needed anymore.

Reply:
Nope, good catch, should be fixed!